.github/workflows/neon_workflow.yml

```
@@ -0,0 +1,93 @@
+    outputs:
+      branch: ${{ steps.branch_name.outputs.current_branch }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get branch name
+        id: branch_name
+        uses: tj-actions/branch-names@v8
+
+  create_neon_branch:
+    name: Create Neon Branch
+    outputs:
+      db_url: ${{ steps.create_neon_branch_encode.outputs.db_url }}
```

actionlint (1.7.4)

property "create_neon_branch_encode" is not defined in object type {create_neon_branch: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)